lib/takeaway.rb

```diff
@@ -0,0 +1,14 @@
+class Takeaway
```

Comment:
Seems to be only printing the menu. Duplicate of functionality in Menu class, and also naming is confusing. I wasn't sure what this did until I looked at the code!